uportal-war/src/main/webapp/WEB-INF/flows/edit-account/editLocalAccount.jsp

```diff
@@ -62,44 +62,73 @@
             <!-- Portlet Section -->
             <div class="portlet-section" role="region">
                 <div class="content">
-
                     <table class="portlet-table table table-hover">
```

Comment:
One second thought the `<fieldset>` tag may not be a good idea either.
Usually the `<fieldset>` tag is use to represent fields that are tightly coupled together.
The `<input>` tags here loosely grouped.
https://accessibility.blog.gov.uk/2016/07/22/using-the-fieldset-and-legend-elements/